frontend/pages/policies/PolicyPage/components/SaveNewPolicyModal/SaveNewPolicyModal.tsx

```diff
@@ -122,6 +122,7 @@ const SaveNewPolicyModal = ({
             label="Name"
             placeholder="What yes or no question does your policy ask about your devices?"
             autofocus
+            ignore1password
```

Comment:
Is this related to this PR or a "let's fix it while we're here" thingy?

Reply:
It's so weird, this was already merged here so I don't know why it's showing up as a new change: https://github.com/fleetdm/fleet/pull/13914/files

Reply:
Confirmed it's in main already: https://github.com/fleetdm/fleet/blob/main/frontend/pages/policies/PolicyPage/components/SaveNewPolicyModal/SaveNewPolicyModal.tsx

I don't want to remove it in case it accidentally reverts to removing it from main